docs/source/system_reference_guide/custom-environments.ipynb

```diff
@@ -38,16 +38,18 @@
     "\n",
     "### Package manager\n",
     "\n",
-    "We use `conda` with the libmamba solver as a package manager to install, update or remove packages (libraries). `conda` works with 'environments' that are directories in your local file system containing a set of packages. When you work 'in a given environment', it means that your programs will look for dependencies in that environment's `conda` directory. All workspaces launch with an environment called `base`, which is a `conda` environment that has all the pre-installed libraries. If you open a terminal launcher after creating a `Basic Stable` workspace : \n",
+    "We use `conda` with the libmamba solver as a package manager to install, update or remove packages (libraries). `conda` works with 'environments' that are directories in your local file system containing a set of packages. When you work 'in a given environment', it means that your programs will look for dependencies in that environment's `conda` directory. All workspaces launch with an initial environment that has all the pre-installed libraries for that workspace. The actual name of that environment depends on the workspace chosen by the user. For example, for the pangeo workspace, the base environment is called `pangeo`. If you open a terminal launcher after creating a `Pangeo` workspace : \n",
     "\n",
     "\n",
-    "![Base conda environment location](../_static/base_conda_environment_location.png)\n",
+    "![Pangeo conda environment location](../_static/pangeo_environment_location.png)\n",
     "\n",
-    "You can notice that a `base` `conda` environment is activated, and its libraries are located in `/opt/conda`. \n",
+    "You can notice that a `pangeo` `conda` environment is activated, and its libraries are located in `/opt/conda/envs/pangeo`. \n",
     "\n",
-    "### Extending the `base` environment in a given workspace session.\n",
+    "### Extending the initial environment in a given workspace session.\n",
     "\n",
-    "**Note : any modification to the `base` environment does not survive a workspace restart. In other words, modifications to `/opt/conda` disappear after a workspace restart**. \n",
+    "**Note 1: hereafter in the documentation the conda environment activated by default for a given workspace is named `base` although it may actually be different (e.g. for pangeo workspaces the name is `pangeo`)**\n",
```

Comment:
This wording confuses me, maybe revise to 
"Note 1: hereafter in the documentation the conda environment activated by default for a given workspace is referred to as `base` although the actual conda environment activated by default depends on the workspace (e.g. for pangeo workspaces the name is `pangeo`)"

Reply:
Yes, I agree